Simplenote/SPEditorTextView+Simplenote.swift

```diff
@@ -30,3 +30,12 @@ extension SPEditorTextView {
         UIPasteboard.general.string = text
     }
 }
+
+extension SPEditorTextView {
+    /// Bounding rect for a character range
+    ///
+    func boundingRectForCharacterRange(_ range: NSRange) -> CGRect {
+        let glyphRange = layoutManager.glyphRange(forCharacterRange: range, actualCharacterRange: nil)
+        return layoutManager.boundingRect(forGlyphRange: glyphRange, in: textContainer)
```

Comment:
Perhaps this [API could be of use?](https://github.com/Automattic/simplenote-ios/blob/develop/Simplenote/Classes/UITextView+Simplenote.swift#L148)?

(It's available in develop!)

Reply:
Thanks, will use it!